Branch.podspec

```diff
@@ -23,5 +23,5 @@ Use the Branch SDK (branch.io) to create and power the links that point back to
   s.requires_arc = true
 
   s.source_files = "Branch-SDK/Branch-SDK/*.{h,m}"
-  s.frameworks = 'AdSupport', 'CoreTelephony'
+  s.frameworks = 'AdSupport', 'CoreTelephony', 'CoreSpotlight', 'MobileCoreServices'
```

Comment:
I'm not sure yet what to do with this. When I run against non-beta Xcode, this doesn't seem to have any effect. I would need to review this before merging, though.